t3f/ops.py

```diff
@@ -751,7 +751,11 @@ def multiply(tt_left, right):
   if not isinstance(right, TensorTrainBase):
     # Assume right is a number, not TensorTrain.
     tt_cores = list(tt_left.tt_cores)
-    tt_cores[0] = right * tt_cores[0]
+    fact = tf.pow(tf.cast(tf.abs(right), tt_left.dtype), 1.0/(tt_left.ndims()))
```

Comment:
I'm not sure if it's a good idea to slow down the multiplication.. Maybe add a flag?

Reply:
Also please add spaces around '/' and add a comment about that's happening here